app/messaging/application/submit-claim.js

```diff
@@ -26,12 +26,23 @@ const submitClaim = async (message) => {
       if (application.dataValues.claimed) {
         return sendMessage({ state: alreadyClaimed }, submitClaimResponseMsgType, applicationResponseQueue, { sessionId: message.sessionId })
       }
-      const res = await updateByReference({ reference, claimed: true, statusId: 4, updatedBy: 'admin', data })
+
+      const applicaitonsCount = await getApplicationsCount()
+
+      let statusId = 9
+      if (applicaitonsCount % 5 === 0) {
```

Comment:
Joy mentioned that we may need to quickly change the frequency of the compliance check. Might be good to extract the value into config via env variable? 

Might also be good for some logging so we know when we have picked an application for a check

Reply:
might be good to wrap this functionality into a new function so it can describe what it is doing and that this logic is for the compliance checks. Might not be obvious what this condition is doing otherwise